.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "dreammaker",
+			"dme": "nebula.dme",
+			"problemMatcher": [
+				"$dreammaker"
+			],
+			"group": {
+				"kind": "build",
+				"isDefault": true
+			},
+			"label": "dm: build - nebula.dme"
+		}
+	]
+}
```

Comment:
Is there any reason for including a file that's automatically generated, and interferes with people with a custom `tasks.json` file?

Reply:
Not sure but kinda yes. I lost mine one time and unable to initialize new one. This one makes shift-ctrl execute build compile by default without pressing what kind build to do.

Reply:
Decided to not include it.